xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-api/src/main/java/org/xwiki/notifications/internal/DefaultNotificationRenderer.java

```diff
@@ -43,6 +44,7 @@
 public class DefaultNotificationRenderer implements NotificationRenderer
 {
     @Inject
+    @Named("wiki")
```

Comment:
Should be context IMO. There is no reason to limit to wikis from what I see in the code.